Sources/SwiftGodot/MacroDefs.swift

```diff
@@ -5,7 +5,7 @@
 //  Created by Miguel de Icaza on 9/25/23.
 //
 
-#if !(os(Windows) && swift(>=5.9))
+#if !(os(Windows) && swift(<5.9))
```

Comment:
This will still leave the macros enabled for swift 5.9 which doesn't have macro support. It should be <=5.9.